### ISSUE TITLE
Alfa Slab One kerning check

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1043]
![screen shot 2017-06-12 at 19 19 43](https://user-images.githubusercontent.com/261579/27059392-35151a6e-4fa4-11e7-9d3c-b9bdbfb80481.png)

**Comment by kalapi** [comment]
@davelab6 This is the old version of Alfa Slab. The kerning is fixed in the latest build (so is that 's').

![screen shot 2017-06-13 at 10 05 29 am](https://user-images.githubusercontent.com/15804460/27066184-e0cfd4cc-501f-11e7-866b-8cf2880d7dac.png)

**Comment by davelab6** [comment]
Thanks Kalapi!! Marc, has this been PR'd?

**Comment by m4rc1e** [comment]
I just downloaded and tested Alfa Slab One off GF. I couldn't replicate this issue. The latest version of the font is v2.000, this is currently on GF as well :-)

![alfa](https://user-images.githubusercontent.com/7525512/27073384-90827abe-501b-11e7-94c8-c15de500ecd8.gif)

I'm more concerned that the .glyphs file crashes the latest version of glyphs. I've filed an [issue](https://forum.glyphsapp.com/t/app-crashes-whilst-attempting-to-load-alfa-slab/6638)
